backend/src/main/java/com/htecgroup/skynest/model/jwtObject/JwtObject.java

```diff
@@ -0,0 +1,11 @@
+package com.htecgroup.skynest.model.jwtObject;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
```

Comment:
Instead of @Data, use just those that are needed

Reply:
Will fix it.